testChanges.sh

```diff
@@ -0,0 +1,7 @@
+cargo build --locked --no-default-features --all
+cargo build --locked --all
+cargo build --locked --features metrics_process --all
+cargo fmt
```

Comment:
```suggestion
```

Reply:
Why are we removing this command? It will automatically format all the files, and thus maintain contain consistent formatting across the repository.

Reply:
My bad, it's OK to leave it.